package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "linter-harbour",
   "main": "./lib/main",
-  "version": "6.6.1",
+  "version": "6.6.2",
```

Comment:
Any version number change will be done when the release is made, please remove this.